internal/runtime/runtime.go

```diff
@@ -27,15 +28,19 @@ import (
 	"github.com/ugorji/go/codec"
 )
 
+// AppFunction type alias
+type AppFunction = func(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{})
+
 // GolangRuntime represents the golang runtime environment
 type GolangRuntime struct {
-	Transforms []func(*appcontext.Context, ...interface{}) (bool, interface{})
+	transforms    []AppFunction
+	IsBusyCopying sync.Mutex
```

Comment:
It doesn't look to me like this needs to be exported

Reply:
yup, this change is un-exporting it

Reply:
yup, thanks!